policies/test.rego

```diff
@@ -21,8 +21,20 @@ deny[{"msg": msg}] {
 
 # Check if all tests succeeded 
 deny[{"msg": msg}] {
-	all_failed := [failure | data.test[_].result != "SUCCESS"; failure := 1]
-	count(all_failed) > 0
+	# Collect all failed tests and convert their name to "test:<name>" format
+	# Reminder: the tests reside in $DATA_DIR/test/<name>/data.json
+	all_failed := {failure | data.test[name].result != "SUCCESS"; failure := sprintf("test:%s", [name])}
+
+	# For the complement operation below (subtraction) we need
+	# non_blocking_checks as set and this creates that from the array
+	non_blocking_set = {x | x := data.config.policy.non_blocking_checks[_]}
+
+	# Failed tests are those that don't have their result equal to "SUCCESS"
+	# and are not on the list of non_blocking_checks
+	failed_non_blocking := all_failed - non_blocking_set
```

Comment:
I suppose this is failed_blocking, right?

Reply:
Tots